### PR TITLE
Major Update - Python 3.6 Compatible + Various API Changes

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**
I merged in this PR the work of @cprieto PR #6 and @viktorbengtsson PR #5.
The plugin API has been changed in order to allow more flexibility.

Thanks to the work of @viktorbengtsson, it solves issue #4 

Part of the work has been inspired by work done by @deepakrb on https://github.com/deepakrb/Pelican-Read-Time.

This PR aims to merge the plugins taking the best out of the two worlds.

Documentation has been updated to allow you to test and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+]
+```
+
+#### 1. Words Per Minute Only
+
+In your settings you would use assign the `READTIME_LANGUAGE_SUPPORT` variable to an integer like so:
+
+***pelicanconf.py***
+```python
+READTIME_LANGUAGE_SUPPORT = 180
```
Comment (deepakrb) [inline README.md]: `READTIME_LANGUAGE_SUPPORT` seems a bit ambiguous, maybe something like `READTIME_WPM`?

File: readtime.py [hunk 2]
```diff
@@ -1,99 +1,159 @@
...
-        Tuple containing minutes and seconds
-    """
-    minutes = int(read_time_seconds / SECONDS_IN_MINUTE)
-    seconds = int(read_time_seconds % SECONDS_IN_MINUTE)
-    return minutes, seconds
+                        if "wpm" not in settings_wpm[key]:
+                            raise Exception("Missing wpm value for the language: %s" % key)
 
+                        if not isinstance(settings_wpm[key]['wpm'], int):
+                            raise Exception("WPM is not an integer for the language: %s" % key)
```
Comment (deepakrb) [inline readtime.py]: There's a mix of two python string formatting `%s` vs `'{}'.format()`. Maybe one would be better?

**Comment by deepakrb** [comment]
Cheers for taking a look at my repo @DEKHTIARJonathan it'd be great if it could end up somewhere since it didn't end up on the main pelican plugins repo 👍

File: README.md (hunk 1 above)
Comment (DEKHTIARJonathan) [reply README.md]: I have changed according to your suggestion, I will commit ASAP ;)

**Comment by DEKHTIARJonathan** [comment]
@deepakrb your remarks have been added in the commit 63531cb.
You are very welcome, you made a very nice job. I just took a lot of inspiration from your work.

**Comment by DEKHTIARJonathan** [comment]
I just modified and improved the *setup.py* file.
I also updated the tests.py to be working with the new API

@JenkinsDev maybe it would be good after merging this PR to submit the project to the official pelican-plugin repository

**Pull request merged**